pipeline/load.py

```diff
@@ -147,7 +147,7 @@ def add_to_article_version_table(conn: connection, df: pd.DataFrame) -> None:
 
 def load_data():
     """Complete data loading in one function. Used for main.py"""
-    db_conn = get_db_connection()
+    db_conn = get_connected_locally()
```

Comment:
will remove this file, must have an older version

Reply:
yeaaa